test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInMetricsListenerTests.cs

```diff
@@ -56,8 +57,11 @@ public void HttpDurationMetricIsEmitted()
             .Build();
 
         var activity = ActivityHelper.StartAspNetActivity(Propagators.DefaultTextMapPropagator, HttpContext.Current, TelemetryHttpModule.Options.OnRequestStartedCallback);
+        Thread.Sleep(1); // Make sure duration is always greater than 0 to avoid flakiness.
         ActivityHelper.StopAspNetActivity(Propagators.DefaultTextMapPropagator, activity, HttpContext.Current, TelemetryHttpModule.Options.OnRequestStoppedCallback);
 
+        Assert.True(duration > 0, "Metric duration should be set.");
```

Comment:
please add this check [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/a372bc065452a7c13bcc69a021e842be440cce13/test/OpenTelemetry.Instrumentation.AspNet.Tests/HttpInMetricsListenerTests.cs#L79) as well.

Reply:
Not exactly sure what you mean since it's the same test method. I ended up moving the check next to the other duration check.